src/App.jsx

```diff
@@ -78,46 +82,59 @@ function App() {
     }
     window.addEventListener('deviceorientation', handleMotionEvent, true);
 
-    if (isMobile) {
-      console.log('Adding Listners');
-      const notes = document.getElementById('notes');
 
-      console.log(notes);
-      // Event Listener for clicking "on" notes
-      notes.addEventListener('touchstart', (e) => {
-        // Grabs note name from 'data-note'
-        console.log(e.touches.length);
-        try {
-          // synth.triggerAttack(e.target.innerText, '16n');
-          console.log('ran');
-          navigator.vibrate(10000);
-          const playNote = getNote(e.target.innerText, rotVal);
-          synth.triggerAttack(playNote, Tone.context.currentTime);
-          console.log('Initial Trigger: ', e.target.innerText, ' Final Trigger: ', playNote);
-        } catch (err) {
-          console.log(err);
+    const playNote = (e) => {
+      // Grabs note name from 'data-note'
+      console.log(e.touches.length);
+      try {
+        // synth.triggerAttack(e.target.innerText, '16n');
+        console.log('ran');
+        navigator.vibrate(10000);
+        if (synthTypeRef.current === 'drums') {
+          playDrums(e);
         }
-      });
+        else {
+          getNote(e.target.innerText, rotVal);
+        }
+      } catch (err) {
+        console.log(err);
+      }
+    }
 
-      // notes.addEventListener('touchmove', (e) => {
-      //   if (e.targetTouches.length === 1) {
-      //     var touch = e.targetTouches[0];
-      //     var x = e.touches[0].clientX;
-      //     var y = e.touches[0].clientY;
-      //     // synth.triggerAttackRelease(["C3", "E3", "G3"], "8n");
-      //     // updateSynth(new Tone.Synth())
-      //   }
-      // })
+    const playDrums = (e) => {
+      const loopLinks = {
+        'loop 1': 'https://tonejs.github.io/audio/drum-samples/handdrum-loop.mp3',
+        'loop 2': 'https://tonejs.github.io/audio/drum-samples/breakbeat.mp3'
+      }
+      if (e.target.innerText === 'loop 1' || e.target.innerText === 'loop 2' || e.target.innerText === 'loop 3' || e.target.innerText === 'loop 4') {
+        console.log(loopLinks[e.target.innerText])
+        const player = new Tone.Player(loopLinks[e.target.innerText]).toDestination();
+        player.loop = true;
+        player.autostart = true;
+      }
+      else {
+        const drum = new Tone.Player(`https://oramics.github.io/sampled/DM/CR-78/samples/${e.target.innerText}.wav`).toDestination();
```

Comment:
Don't link to binary files from a user's GitHub repo, put them in the project repo along with other static files 
Vercel serves files under 5mb  @codevaam

Reply:
PWA should run offline without external dependencies. Having files in the app itself will help.

Reply:
Made the changes, check once and merge